plugins/modules/proxysql_info.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: proxysql_info
+author: "Markus Bergholz (@markuman)"
+short_description: Gathers information about proxysql server
+description:
+   - Gathers information about proxysql server.
+version_added: '1.2.0'
+extends_documentation_fragment:
+- community.proxysql.proxysql.connectivity
+notes:
+- Supports C(check_mode).
+'''
+
+EXAMPLES = '''
+---
+# This example adds a rule for fast routing
+- name: Add a rule
```

Comment:
```suggestion
# This example adds a rule for fast routing
- name: Add a rule
```
Ah, I see that this is an early draft. OK, when it's ready for review, please let me know